src/Month.js

```diff
@@ -89,7 +90,13 @@ class MonthView extends React.Component {
           scrollableMonth && 'rbc-month-view-infinite'
         )}
       >
-        <div className="rbc-row rbc-month-header">
+        <div
+          className={'rbc-row rbc-month-header'}
```

Comment:
Could we change this back to "rbc-row rbc-month-header" since we don't need the curly brackets